dbgv/dbgv_benchmark_test.go

```diff
@@ -0,0 +1,291 @@
+package dbgv
+
+import (
+	"testing"
+
+	"github.com/tuneinsight/lattigo/v3/bgv"
+	"github.com/tuneinsight/lattigo/v3/drlwe"
+	"github.com/tuneinsight/lattigo/v3/rlwe"
+)
+
+func Benchmark_Dbgv(b *testing.B) {
+
+	var err error
+	for _, p := range TestParams[:] {
+		var params bgv.Parameters
+		if params, err = bgv.NewParametersFromLiteral(p); err != nil {
+			b.Fatal(err)
+		}
+
+		var tc *testContext
+		if tc, err = gentestContext(params); err != nil {
+			b.Fatal(err)
+		}
+
+		benchPublicKeyGen(tc, b)
```

Comment:
The keygen-related benchmarks will be added by `dev_threshold` at the `rlwe` layer. They can be removed.

Reply:
Done